aggregator/log_manager.py

```diff
@@ -60,14 +60,14 @@ def get_reads(self, start_timestamp = None, end_timestamp = None):
       end_timestamp = self.end_time if self.end_time is not None else int(time.time())
     self.cursor.execute('SELECT * FROM Log WHERE request_type = \'READ\' AND status = 200 AND timestamp >= ? AND timestamp <= ? ORDER BY timestamp ASC', (start_timestamp, end_timestamp))
     return self.cursor.fetchall()
-  
+
   # Retrieve successful log on file movement in a specified time period
   def get_movings(self, start_timestamp = None, end_timestamp = None):
     if start_timestamp == None:
       start_timestamp = self.start_time if self.start_time is not None else 0
     if end_timestamp == None:
       end_timestamp = self.end_time if self.end_time is not None else int(time.time())
-    self.cursor.execute('SELECT * FROM Log WHERE (request_type = \'TRANSFER\' OR request_type = \'REPLICATE\' OR request_type = \'DISTRIBUTED_REPLICATE\') AND status = 200 AND timestamp >= ? AND timestamp <= ? ORDER BY timestamp ASC', (start_timestamp, end_timestamp))
+    self.cursor.execute('SELECT * FROM Log WHERE (request_type = \'TRANSFER\' OR request_type = \'REPLICATE\' OR request_type = \'DISTRIBUTED_REPLICATE\' OR request_type = \'DISTRIBUTED_GREEDY_REPLICATION\') AND status = 200 AND timestamp >= ? AND timestamp <= ? ORDER BY timestamp ASC', (start_timestamp, end_timestamp))
```

Comment:
Does it need its own name, or could we just stick with `DISTRIBUTED_REPLICATE`?

What if.. we just had `TRANSFER` and `REPLICATE`, and in the logs, we made the `source_entity` to be the name of the algorithm? e.g. `volley` or `distributed_replicate` or `central_greedy` or `dist_greedy`, etc.?

Just an idea